src/browser/setupWorker/glossary.ts

```diff
@@ -195,6 +195,11 @@ export interface StartOptions extends SharedOptions {
    * of all registered Service Workers on the page.
    */
   findWorker?: FindWorker
+
+  /**
+   * A flag to enable MSW in production (NODE_ENV === 'production'), otherwise it will throw an error
+   */
+  dangerouslyRunInProduction?: boolean
```

Comment:
`StartOptions` here extends `SharedOptions` so there's no need to add this new property here as well.